packages/assets-controllers/src/NftController.ts

```diff
@@ -1965,6 +1994,27 @@ export class NftController extends BaseController<
     );
   }
 
+  /**
+   * Fetches NFT Collection Metadata from the NFT API.
+   *
+   * @param chainIds - An array of chain IDs to query
+   * @param tokens - The array of objects containing contract addresses and tokenIds.
+   * @returns NFT tokens including collection info.
+   */
+  async getNFTTokenInfo(
+    chainIds: Hex[],
+    tokens: {
+      contractAddress: string;
+      tokenId: string;
```

Comment:
For the confirmations use case, why do we need the token ID, since are primary goal is to extract whether a specific contract address has `isSpam` set?